### PR TITLE
Skip sbt to fix Snyk job

### PR DESCRIPTION
**Pull request by ioannakok**
Fixing Snyk job!

### CODE REVIEW

File: .github/workflows/snyk.yml [hunk 1]
```diff
@@ -17,5 +17,6 @@ jobs:
       DEBUG: true
       ORG: guardian-mobile
       SKIP_NODE: false
+      SKIP_SBT: true
```
Comment (ioannakok) [inline .github/workflows/snyk.yml:20]: Last Snyk job [failed in main ](https://github.com/guardian/react-crossword/actions/runs/4345653064/jobs/7590665642)with error: 

```
Error: No file in /home/runner/work/react-crossword/react-crossword matched to [**/*.sbt,**/project/build.properties,**/project/**.{scala,sbt}], make sure you have checked out the target repository
```

Found this flag [in the docs](https://github.com/guardian/.github/blob/bf210f13ef604faa8682a0e7bc17113983470303/.github/workflows/docs/snyk.md#skip-sbt) of this workflow.

Comment (ioannakok) [reply .github/workflows/snyk.yml:20]: Triggered the job manually in this branch and it [succeeded](https://github.com/guardian/react-crossword/actions/runs/4345844339).

Comment (DavidLawes) [reply .github/workflows/snyk.yml:20]: Thanks - great spot!

**Review by DavidLawes — APPROVED** [review]
👍

**Pull request merged**